docs/hints-for-use-with-docker.md

```diff
@@ -8,7 +8,7 @@
 docker run \
   -v $PWD/:/project \ # Mount current working directory into /project to use as input.
   ort --info \
-  -c /project/ort/config.hocon \ # Use file from "<workingdirectory>/ort" as config.
+  -c /project/ort/config.yml \ # Use file from "<workingdirectory>/ort" as config.
```

Comment:
Huh, interesting, the original `config.hocon` file was already wrong. Good that you found this!